libbpf-rs/tests/test.rs

```diff
@@ -370,6 +370,55 @@ fn test_sudo_object_map_empty_lookup() {
         .is_none());
 }
 
+#[test]
+fn test_sudo_object_map_queue_crud() {
```

Comment:
Let's add a summary comment what the test does?

Reply:
agree with that, I extended the test to cover the stack, with comment.